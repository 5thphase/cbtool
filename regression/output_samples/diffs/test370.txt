hpcc, coremark, iperf, redis_ycsb, nullworkload, ibmderby_tradelite, ddgen, windesktop, hadoop, mongo_ycsb, cassandra_ycsb, giraph, mweb, fbnfsrd, filebench, specjbb, specimap, mlg, ibm_daytrader, netperf, open_daytrader, ibmdb2_tradelite
