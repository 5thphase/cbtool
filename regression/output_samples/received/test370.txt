AIs with the following types can be attached to to this experiment (Cloud TESTCLOUD) :
nullworkload, ddgen, specjbb, fbnfsrd, lost, mlg, windesktop, mlg2, netperf, hadoop, hpcc, mongo_ycsb, daytrader, cassandra_ycsb, coremark, tradelite, specimap, mweb, specweb, filebench
