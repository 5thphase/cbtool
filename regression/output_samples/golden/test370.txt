AIs with the following types can be attached to to this experiment (Cloud TESTCLOUD) :
nullworkload, fbnfsrd, ddgen, mlg2, filebench, lost, windesktop, netperf, hadoop, hpcc, mongo_ycsb, daytrader, coremark, tradelite, specimap, mweb, specweb, specjbb, mlg
